tests/release/const.go

```diff
@@ -1,22 +1 @@
 package release
-
-// import (
-// 	"time"
-// )
-
-// const (
```

Comment:
removing these doesn't break anything? why did we have them there then?

Reply:
In full tests we needed them

Reply:
Then why are we removing them? Isn't the goal to get the full tests turned back on?

Reply:
Its just a side test for namespace creation & deletion
The test are currently being worked over to enable them, its a side story

Reply:
If it is a side test, why remove these as part of this PR? Why not just leave them as they are? The PR to re-enable them can modify them as needed

Reply:
This comment is my only left before approving

Reply:
In the https://github.com/redhat-appstudio/e2e-tests/ repo these files are removed to allow the infra-deployment  CI process pass
Until the e2e-tests are fully functional these files are not there
![image](https://user-images.githubusercontent.com/640645/190403388-413cebf5-915a-41c1-bae5-4fe8bc95d0b4.png)

Reply:
You are right, returned the const.go file to its state after release tests where disabled to allow infra-deployment CI to pass

Reply:
Maybe something wrong with the rebase? I still see const.go showing up in files changed

Reply:
returned the const.go file to its state after release tests where disabled to allow infra-deployment CI to pass